libs/partners/mistralai/langchain_mistralai/embeddings.py

```diff
@@ -215,16 +220,27 @@ def embed_documents(self, texts: List[str]) -> List[List[float]]:
             List of embeddings, one for each text.
         """
         try:
-            batch_responses = (
-                self.client.post(
+            batch_responses = []
+
+            @retry(
+                retry=retry_if_exception_type(httpx.TimeoutException),
```

Comment:
(no need to change if you don't want) This is OK b/c it's probably the dominant failure mode

But it's very common to retry 5xx errors as well. And 408 (request timeout)